.github/workflows/acceptance.yml

```diff
@@ -14,7 +22,39 @@ on:
   pull_request:
 
 jobs:
+  changes:
+    runs-on: ubuntu-18.04
+    # Set job outputs to values from filter step
+    outputs:
+      backend_or_frontend: ${{ steps.filter.outputs.backend_or_frontend }}
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout sentry
+
+      - name: Check for FE or BE file changes
+        uses: getsentry/paths-filter@v2
+        id: filter
+        with:
+          token: ${{ github.token }}
+          filters: .github/file-filters.yml
+
+  # It is necessary to update the name and matrix in tandem with the frontend workflow
+  # to meet the required check
+  fake-frontend:
```

Comment:
For every required check, we will have a fake workflow that uses the same matrix.

**NOTE**: The following screenshots are from a [prior commit](https://github.com/getsentry/sentry/actions/runs/2004351521) that pretends to have **no backend and no frontend changes**. The latest commit will have all checks passing and the fake ones will be skipped. See the following the diff to understand what I mean:

```diff
- '.github/workflows/!(js-*)'
+ # Temp change
+ '.github/workflows/!(acceptance|visual-diff).yml'
```

This shows how the fake workflows are satisfying the "Required" requirement:
<img width="726" alt="image" src="https://user-images.githubusercontent.com/44410/159011739-31c14812-74bf-408c-a180-acfae8b604ad.png">

You might be surprised but this approach (instead of adding and if condition to every step) is [the recommended one by Github](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks) for required checks when they get skipped.

Screenshot of meeting the Visual Snapshot requirement:
<img width="726" alt="image" src="https://user-images.githubusercontent.com/44410/159012132-10940bf1-f5d4-4018-b11e-792224daf65f.png">

Once this PR merges to master we would *not* see this check triggered when there are no FE or BE changes. We're currently seeing it because "on workflow_run" is influenced by what is checked in on `master` rather than on this PR:
<img width="350" alt="image" src="https://user-images.githubusercontent.com/44410/159011887-13df6e8c-766d-4533-9eff-2f812846be42.png">

```
name: visual diff
on:
  workflow_run:
    workflows:
      - acceptance
    types:
      - completed
```